browser/extensions/brave_extension_provider_browsertest.cc

```diff
@@ -18,7 +18,11 @@
 #include "chrome/test/base/ui_test_utils.h"
 #include "content/public/test/browser_test_utils.h"
 
-using BraveExtensionProviderTest = extensions::ExtensionFunctionalTest;
+class BraveExtensionProviderTest : public extensions::ExtensionFunctionalTest {
+ public:
+  void SetUpOnMainThread() override {
+    extensions::ExtensionFunctionalTest::SetUpOnMainThread();
+};
```

Comment:
Missing `}`. Fixed by `7c04b4c`, but not sure if this was meant to have more code.

Reply:
Sorry, there were a lot of last-minute changes.